File: Problem_Set_01/mario/less/mario.c
#include <stdio.h>
#include <cs50.h>

int main(void)
{
    int t;
    do
    {
        t = get_int("Height: ");
    }
    while (t>8||t<1);

    for(int i=0;i<t;i++)
    {
       for(int j= t-1; j>i;j = j-1)
            {
                printf(" ");
            }
       for(int k=-1;k<i;k++)
        {

            printf("#");
        }
        printf("\n");
    }
}

File: Problem_Set_06/sentimental/mario/less/mario.py
from cs50 import get_int

i = 0
while (i < 1 or i > 8):
    i = get_int(" ")

bpc = i
hashw = 0
for i in range(i):
    bpc -= 1
    hashw += 1
    print(" " * bpc, end="")
    print("#" * hashw, end="")
    print("")
